### ISSUE TITLE
Electron: support vibrance module and GTK styling

### ISSUE THREAD

**Issue by DaniellMesquita** [keeweb/keeweb#1261]

**Comment by antelle** [comment]
Hi! Please give it more description: what is it?

**Comment by DaniellMesquita** [comment]
Vibrancy supports glass effects on Windows and Mac - https://github.com/arkenthera/electron-vibrancy
GTK is the most popular theming engine, and supports SVG and CSS - https://www.npmjs.com/package/electron-gtk-theme

**Comment by DaniellMesquita** [comment]
KeeWeb could support both. On Windows and Mac, it supports the glass theming.
On Linux, it integrates with any GTK theme, as KeePassX integrates with QT themes.

**Comment by antelle** [comment]
I don't think it will work well with KeeWeb themes, there are quite a lot of customizations here.

**Comment by DaniellMesquita** [comment]
Alongside KeeWeb themes, it can have the "system/OS" theme option as default.

**Comment by antelle** [comment]
It can, but it also needs some styles, and those styles must be somehow picked so that we have theme styles separated from them, which is a huge amount of work.
